src/app/index.html

```diff
@@ -1,15 +1,8 @@
-<sky-fluid-grid>
-  <sky-row>
-    <sky-column>
-      <h1>
-        Demos
-      </h1>
-
-      <ul>
-        <li>
-          <a href="#" skyAppLink="/demos/sample">Sample</a>
-        </li>
-      </ul>
-    </sky-column>
-  </sky-row>
-</sky-fluid-grid>
+<ul>
+  <li>
+    <a href="#" skyAppLink="/visual/checkbox">Checkbox</a>
+  </li>
+  <li>
+    <a href="#" skyAppLink="/visual/radio">Radio</a>
+  </li>
```

Comment:
We need to add File Attachments to this menu as well.

Reply:
done. good catch.